src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -325,10 +325,9 @@
                 (when (seq stack-trace)
                   [StackTraceViewer {:lines stack-trace}])]]
               [:div {}
-               (style/create-link {:text [:span {}
-                                          (icons/icon {:className "fa-fw"} :disclosure-closed)
-                                          "Show Details"]
-                                   :onClick #(swap! state assoc :expanded? true)})])]))))})
+               (links/create-internal {:onClick #(swap! state assoc :expanded? true)}
```

Comment:
We have other things like this. If there was an easy way to set them all, I'd be in favor of it.